index.android.js

```diff
@@ -2,7 +2,7 @@
  * @providesModule LinearGradient
  * @flow
  */
-import React, { Component } from 'react';
+import React, { Component, createRef } from 'react'
```

Comment:
Minor: Looks like you accidentally removed the semicolon on this line.

Reply:
Fixed, thanks